studio/components/interfaces/Organization/IntegrationSettings/Integration.tsx

```diff
@@ -158,3 +110,71 @@ Repository connections for ${title?.toLowerCase()}
 }
 
 export default Integration
+
+type IntegrationConnectionItemProps = {
+  connection: IntegrationProjectConnection
+  type: IntegrationName
+  onDeleteConnection: (connection: IntegrationProjectConnection) => void | Promise<void>
+}
+
+const IntegrationConnectionItem = ({
+  connection,
+  type,
+  onDeleteConnection,
+}: IntegrationConnectionItemProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const onConfirm = useCallback(async () => {
+    try {
+      await onDeleteConnection(connection)
+    } finally {
+      setIsOpen(false)
+    }
+  }, [connection, onDeleteConnection])
+
+  const onCancel = useCallback(() => {
+    setIsOpen(false)
+  }, [])
+
+  return (
+    <>
+      <IntegrationConnection
+        connection={connection}
+        type={type}
+        actions={
+          <Dropdown
+            side="bottom"
+            align="end"
+            size="large"
+            overlay={
+              <>
+                <Dropdown.Item icon={<IconTrash size={14} />} onSelect={() => setIsOpen(true)}>
+                  Delete
+                </Dropdown.Item>
+              </>
+            }
+          >
+            <Button asChild iconRight={<IconChevronDown />} type="default">
+              <span>Manage</span>
+            </Button>
+          </Dropdown>
+        }
+      />
+
+      <ConfirmationModal
+        visible={isOpen}
+        danger={true}
+        header="Confirm to delete"
+        buttonLabel="Delete"
+        onSelectCancel={onCancel}
+        onSelectConfirm={onConfirm}
+      >
+        <Modal.Content>
+          <p className="py-4 text-sm text-scale-1100">
```

Comment:
use the new tailwind classes that jonny mentioned
```suggestion
          <p className="py-4 text-sm text-light">
```